vue-components/src/components/DateInput.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<div
+		:class="[ 'wikit', 'wikit-DateInput' ]"
+	>
+		<InputWithExtender
+			:label="label"
+			:value="value"
+			:error="error"
+			:placeholder="placeholder"
+			:disabled="disabled"
+			@input="$emit( 'input', $event )"
+		>
+			<template v-slot:default>
+				<p class="wikit-DateInput__IntroText">
+					{{ resultsIntroText }}
+				</p>
+				<p class="wikit-DateInput__ParsedValue" v-if="parsedValue">
+					<b>{{ parsedValue }}</b>
+				</p>
+				<slot v-if="!value" name="prompt" />
+				<BouncingDots v-if="!parsedValue && value" type="small" />
+				<div class="wikit-DateInput__CalendarNotice" v-if="calendarNotice && parsedValue">
+					<Icon type="info" size="small" />
+					{{ calendarNotice }}
+				</div>
+			</template>
+			<template v-slot:suffix>
+				<slot name="suffix" />
+			</template>
+		</InputWithExtender>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import VueCompositionAPI, { defineComponent } from '@vue/composition-api';
+import { errorProp } from '@/compositions/validatable';
+import Icon from '@/components/Icon.vue';
+import InputWithExtender from '@/components/InputWithExtender.vue';
+import BouncingDots from '@/components/BouncingDots.vue';
+
+Vue.use( VueCompositionAPI );
+
+export default defineComponent( {
+	name: 'DateInput',
+	components: {
+		BouncingDots,
+		Icon,
+		InputWithExtender,
+	},
+	props: {
+		error: errorProp,
+		disabled: {
+			type: Boolean,
+			default: false,
+		},
+		placeholder: {
+			type: String,
+			default: '',
+		},
+		label: {
+			type: String,
+			default: null,
+		},
+		parsedValue: {
+			type: String,
+			default: null,
+		},
+		resultsIntroText: {
+			type: String,
+			default: null,
+		},
+		value: {
+			type: String,
+			default: null,
+		},
+		calendarNotice: {
+			type: String,
+			default: null,
+		},
+	},
+} );
+</script>
+
+<style lang="scss">
+.wikit-DateInput {
+	&__IntroText {
+		font-family: $font-family-style-label;
```

Comment:
```suggestion
		padding-block: $wikit-InputExtender-content-padding-vertical;
		font-family: $font-family-style-label;
```

Reply:
should be added to each one of these, right? 

sorry, I'd usually try this out locally, but I find lots of conflicts that I'm not sure how to resolve when I try to work on this branch locally